src/Illuminate/Queue/WorkerOptions.php

```diff
@@ -74,6 +74,13 @@ class WorkerOptions
      */
     public $maxTime;
 
+    /**
+     * The number of seconds to rest between jobs.
+     *
+     * @var int
```

Comment:
Your example is float?

Reply:
similar to sleep. maybe in another commit we change both to float.